### PR TITLE
update wd monitors after ui change

### PR DESCRIPTION
**Pull request by HugoPct**
### What does this PR do? What is the motivation?
We updated the UI for Watchdog Monitors.
this PR changes the docs to reflect such change
https://app.datadoghq.com/monitors/create/watchdog

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/types/watchdog.md [hunk 1]
```diff
@@ -22,107 +22,32 @@ further_reading:
...
 
-[1]: /monitors/configuration/?tab=thresholdalert#alert-grouping
-[2]: /logs/log_configuration/attributes_naming_convention/#reserved-attributes
-{{% /tab %}}
-{{< /tabs >}}
-
 ### Notifications
 
-For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][3] page.
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][4] page.
```
Comment (estherk15) [inline content/en/monitors/types/watchdog.md:44]:
```suggestion
For more instructions on the **Notify your team** section, see the [Notifications][4] page.
```

File: content/en/monitors/types/watchdog.md [hunk 2]
```diff
@@ -22,107 +22,32 @@ further_reading:
...
+* Environment. These values are derived from the `env` tag.
+* Alert Category. Scope the monitor to a subset of Watchdog alerts.
+* Team. These values are derived from the Service Catalog.
 
-**3. Group By**
-* The dimensions you want to [group notifications by][1]
+**2. Additional scoping**
+* Filter on any additional tag available on the Watchdog event
+* Group By the dimensions you want to [group notifications by][3]
+{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-2.png" alt="Configuring a Watchdog Monitor with advanced settings" style="width:80%;">}}
```
Comment (estherk15) [inline content/en/monitors/types/watchdog.md:38]:
```suggestion
* Filter on any additional tag available on the Watchdog event.
* Group By the dimensions you want to [group notifications by][3].
{{< img src="/monitors/monitor_types/watchdog/watchdog-monitor-2.png" alt="Configuring a Watchdog Monitor with advanced settings" style="width:90%;">}}
```

**Review by estherk15 — CHANGES_REQUESTED** [review]
Looks good, left feedback for image and minor fixes!

**Review by HugoPct — COMMENTED** [review]
accepted suggestions

**Review by estherk15 — APPROVED** [review]

**Pull request merged**